MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -120,7 +118,7 @@ private void FetchDataFromTags(Audio audio)
         {
             var file = TagLib.File.Create(audio.Path);
             var tagTypes = file.TagTypesOnDisk;
-            Tag tags = null;
+            Tag tags = new TagLib.Id3v2.Tag();
```

Comment:
I'd rather you made it nullable
```suggestion
            Tag? tags = null;
```